### DISCUSSION TITLE
11m band (yes I know ...)

### DISCUSSION THREAD

**Discussion by iplayradionetworks** [magicbug/Cloudlog#2070] [General]
I am sure others have might asked for this band to be implemented... 

And I am sure you might want to avoid "headaches"... I was wondering tho if I could install the app on another subdomain, create a new logbook there and and a new API key. 

The reason for that is to be able to use the app with two different apis . Not sure if it's possible tho .

To be able to use the app that way I am not sure tho.

Another scenario is if I could manually edit the sql database via phpmyadmin and add 11m band there or perhaps use 160m band as the 11m band and simplify things ... As I won't be using 160m band anyways....

Any thoughts ?

Jim

**Comment by pa8s** [comment]
> Another scenario is if I could manually edit the sql database via phpmyadmin and add 11m band there or perhaps use 160m band as the 11m band and simplify things ... As I won't be using 160m band anyways....

If I understand your question correctly you want to include the 11m band? This can be done within Cloudlog. Go to your user menu and select the Band submenu. Here you can add, modify or delete bands. So adding 11m should not be a problem and prevents issues while editing in phpmyadmin.

**Reply by iplayradionetworks** [reply]
I didn't know there a possibility to add a new band within the app .....

But on the mobile we have to type it manual I guess ?

**Reply by pa8s** [reply]
Also on the desktop you have fill it in manually. Band limits can vary per country.

**Reply by iplayradionetworks** [reply]
What desktop you mean?

I asked also for the mobile app. 

The desktop is actually the web app ...

**Reply by pa8s** [reply]
With desktop I mean the webpage viewed in desktop mode on a pc or laptop.

**Reply by iplayradionetworks** [reply]
Ok yeap .

So what about the mobile app ? 

We just type the frequency for example 27.285 ? And that's all ?

Do we need to have two different logbooks in the app ? Or just one logbook unified that holds all qsos from amateur+cb radio ?

**Reply by pa8s** [reply]
If you select 'Create a band' you'll get a popup screen (on desktop as on mobile) in which you will enter the requested data.

So band and bandgroup will be '11m' and 'hf', SSQ QRG 27285000 as per your example, frequency in Hz. Data and CW QRG you have to google yourself or set it to a value in the band. Unselect all award ticks as these are not valid for 11m as far as I know.

It is up to you if you want separate logbooks of all in one. I have no idea what will happen when you upload 11m QSOs to e.g. LoTW of QRZ

I use one logbook for when I use my personal call and another when participating in a SES.

Btw, I'm just one of many users of Cloudlog

**Reply by iplayradionetworks** [reply]
On mobile app there is no option to create band .

I am not referring to the mobile version of the web app but the individual Android app for cloudlog.

**Reply by pa8s** [reply]
Ah that changes things. I was under the impression you were referring to the mobile version of the website. I don't know about the Android app.

**Reply by phl0** [reply]
> On mobile app there is no option to create band .
> 
> I am not referring to the mobile version of the web app but the individual Android app for cloudlog.

Are we talking about [CloudLogOffline](https://github.com/myzinsky/cloudLogOffline)? Then there is no such option as to create a band.
